src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsAdornmentTaggerProvider.cs

```diff
@@ -78,9 +78,9 @@ protected object CreateToolTipContent(Workspace workspace, DiagnosticData diagno
         }
 
         protected virtual SnapshotSpan AdjustSnapshotSpan(SnapshotSpan span, int minimumLength)
-            => AdjustSnapshotSpan(span, minimumLength, int.MaxValue);
+            => AbstractDiagnosticsAdornmentTaggerProvider<TTag>.AdjustSnapshotSpan(span, minimumLength, int.MaxValue);
```

Comment:
seems unnecessary.

Reply:
Hmm code fix added it, and I think it adds simplifier annotations. Let me see if there is underlying bug in simplifier.

Reply:
Yep, seems like simplifier is not marking this as unnecessary, hence the behavior. I have manually removed it.